test/1_BscBridge.js

```diff
@@ -133,13 +130,6 @@ describe("BridgeBSC", function () {
         .connect(operator)
         .transferAssetByOperator(invalidAmount, userB.address),
     ).to.be.revertedWith("INVALID_AMOUNT");
-
-    const overflowedAmount = ethers.utils.parseUnits("2000", 18);
-    await expect(
-      bridgeBSC
-        .connect(operator)
-        .transferAssetByOperator(overflowedAmount, userB.address),
-    ).to.be.revertedWith("INSUFFICIENT_BALANCE");
   });
 
   it("Should return total transfer amount correctly", async function () {
```

Comment:
Since the Expect statement removed,
Please check whether this test block is still required.
I think it's better to just delete the test block if it doesn't have the functionality.